R/agg_sums_means.R

```diff
@@ -0,0 +1,50 @@
+#' Aggregate Student Level to School/District/State
+#'
+#' This function takes a student level dataframe and aggregates it to the school/district/state level.
+#' Provides sums and means for desired columns.
+#' @param df Dataframe. Used first to be friendly to the \%>\% operator.
+#' @param sum_cols A character vector of the columns to be summed.
+#' @param mean_cols A character vector of the columns to find the mean. Use empty vector if no desired mean columns.
+#' @param ... The columns to aggregate at (e.g. system, system_name, school, school_name)
+#' @return Returns a dataframe grouped by specified fields.
+#' @keywords aggregate, group, collapse
+#' @examples
+#' agg_student_level(student_level_df, c('enrolled', 'tested', 'valid tests'),
+#' c(), system, system_name, subgroup)
+#' agg_student_level(student_level_df, c('enrolled', 'tested', 'valid tests'),
+#' c(), system, school, subgroup)
+#' @export
+
+
+agg_sums_means <- function(df, sum_cols, mean_cols, ...){
+    if (length(sum_cols) > 0){
```

Comment:
What if `sum_cols` or `mean_cols` is missing? Let's add that so that those arguments are optional.